os-jws-https/added/https.sh

```diff
@@ -17,27 +17,26 @@ function configure() {
 
 function configure_https() {
   https="<!-- No HTTPS configuration discovered -->"
-  if [ -n "${JWS_HTTPS_CERTIFICATE_DIR}" -a -n "${JWS_HTTPS_CERTIFICATE}" -a -n "${JWS_HTTPS_CERTIFICATE_KEY}" ] ; then
-      password=""
-      if [ -n "${JWS_HTTPS_CERTIFICATE_PASSWORD}" ] ; then
-          password=" SSLPassword=\"${JWS_HTTPS_CERTIFICATE_PASSWORD}\" "
-      fi
-      https="<Connector \
-             protocol=\"org.apache.coyote.http11.Http11AprProtocol\" \
-             port=\"8443\" maxThreads=\"200\" \
-             scheme=\"https\" secure=\"true\" SSLEnabled=\"true\" \
-             SSLCertificateFile=\"${JWS_HTTPS_CERTIFICATE_DIR}/${JWS_HTTPS_CERTIFICATE}\" \
-             SSLCertificateKeyFile=\"${JWS_HTTPS_CERTIFICATE_DIR}/${JWS_HTTPS_CERTIFICATE_KEY}\" \
-             ${password}  \
-             SSLVerifyClient=\"optional_no_ca\" SSLProtocol=\"TLSv1.3+TLSv1.2\""
-
-      if [ -n "$JWS_SERVER_NAME" ]; then
-        https="$https server=\"${JWS_SERVER_NAME}\""
-      fi
-
-      https="$https />"
-
-  elif [ -n "${JWS_HTTPS_CERTIFICATE_DIR}" -o -n "${JWS_HTTPS_CERTIFICATE}" -o -n "${JWS_HTTPS_CERTIFICATE_KEY}" ] ; then
+  password=""
+  if [ -n "${JWS_HTTPS_CERTIFICATE_PASSWORD}" ] ; then
+    password=" SSLPassword=\"${JWS_HTTPS_CERTIFICATE_PASSWORD}\" "
+  fi
+  servername=""
+  if [ -n "${JWS_SERVER_NAME}" ]; then
+    https="$https server=\"${JWS_SERVER_NAME}\""
+  fi
+  if [ -n "${JWS_HTTPS_CERTIFICATE_DIR}" -a -n "${JWS_HTTPS_CERTIFICATE}" -a -n "${JWS_HTTPS_CERTIFICATE_KEY}" -a -n "${JWS_HTTPS_CACERTIFICATE}"] ; then
+      https="<Connector port=\"8443\" protocol=\"HTTP/1.1\" maxThreads=\"200\" SSLEnabled=\"true\" ${password} ${servername}> \
+      <SSLHostConfig caCertificateFile=\"${JWS_HTTPS_CERTIFICATE_DIR}/${JWS_HTTPS_CACERTIFICATE}\" certificateVerification=\"optional\"> \
```

Comment:
That is still not quite right:
The  <SSLHostConfig/> needs a certificateKeyPassword not a SSLPassword
In fact that what you have will work, see https://tomcat.apache.org/tomcat-9.0-doc/config/http.html#SSL_Support